examples/tutorials/advanced/panel_extension.py

```diff
@@ -0,0 +1,121 @@
+r"""
+Dynamic data visualization using ``Panel``
+==========================================
+*Please run the following code examples in a notebook environment otherwise
+the interactive parts of this tutorial will not work. You can use the button
+"Download Jupyter notebook: panel_extension.ipynb" at the bottom of this page
+to download this script as a Jupyter notebook.*
+
+The library ``Panel`` (https://panel.holoviz.org/index.html) can be used to
```

Comment:
Since the intersphinx link for `panel` is only used once, I feel it's not necessary to use intersphinx for `panel`.

We can just use
```
`Panel <https://panel.holoviz.org/index.html>`__
```

Reply:
> Since the intersphinx link for `panel` is only used once, I feel it's not necessary to use intersphinx for `panel`.
> 
> We can just use
> 
> ```
> `Panel <https://panel.holoviz.org/index.html>`__
> ```

This is updated in commit https://github.com/GenericMappingTools/pygmt/pull/2498/commits/0f2550858d90326eb6805d965fe27c3151fd2671.